looper/models.py

```diff
@@ -441,6 +441,8 @@ def __init__(self, config_file, subproject=None,
         self.sheet = None
         self.samples = list()
 
+        self.add_sample_sheet()
```

Comment:
And even then it'd only matter if the client developer needed additional code to execute *before* adding the sample sheet